### PR TITLE
fix(website-frontend): background color staying the same after viewing report

### LINKED ISSUE #895: After viewing a report, the body of the website remains in the wrong color
image
image

### PR DESCRIPTION
**Pull request by xandervedder**
closes #895

### CODE REVIEW

File: packages/website-frontend/index.html [hunk 1]
```diff
@@ -2,10 +2,26 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Try out the Stryker dashboard, home for your test quality">
+    <link rel="icon" type="image/svg+xml" href="/src/assets/favicon.ico" />
```
Comment (hugo-vrijswijk) [inline packages/website-frontend/index.html:7]: Shouldn't this link to a favicon in the `public` folder?

File: packages/website-frontend/index.html [hunk 2]
```diff
@@ -2,10 +2,26 @@
...
     <title>Stryker Dashboard</title>
     <script type="module" src="./src/app.ts"></script>
+    <style>
```
Comment (hugo-vrijswijk) [inline packages/website-frontend/index.html:10]: Could you move this css to an imported stylesheet?

File: packages/website-frontend/src/pages/report.page.ts [hunk 3]
```diff
@@ -49,6 +49,12 @@ export class ReportPage extends LitElement {
       });
   }
 
+  #prepareStyling() {
+    this.style.display = 'flex';
```
Comment (hugo-vrijswijk) [inline packages/website-frontend/src/pages/report.page.ts:53]: Can't you set these properties with css? That would be a little cleaner

File: packages/website-frontend/index.html (hunk 1 above)
Comment (xandervedder) [reply packages/website-frontend/index.html:7]: It should, yes 👍

File: packages/website-frontend/README.md [hunk 4]
```diff
@@ -16,3 +16,80 @@ Run `npm run build` to build the project. The build artifacts will be stored in
 ## Running tests
 
 Run `npm run test` to execute the tests via [Vitest](https://vitest.dev/) and [Playwright](https://playwright.dev/).
+
+## Adding a report to the dashboard
+
+If you need to add a report quickly to the report, here is a simple CURL command that uploads one:
```
Comment (hugo-vrijswijk) [inline packages/website-frontend/README.md:22]: You can refer to the dashboard docs: https://stryker-mutator.io/docs/General/dashboard/#send-a-report-via-curl

File: packages/website-frontend/index.html [hunk 5]
```diff
@@ -2,8 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Try out the Stryker dashboard, home for your test quality" />
+    <link rel="icon" type="image/svg+xml" href="/assets/favicon.ico" />
```
Comment (hugo-vrijswijk) [inline packages/website-frontend/index.html:7]: Is this an svg or an ico? The type says svg, but the extension says ico. You can set an svg as the "preferred" favicon, with ico as a fallback:

```html
<link rel="icon" href="/favicon.ico" sizes="48x48">
<link rel="icon" href="/favicon.svg" sizes="any" type="image/svg+xml">
```

Comment (xandervedder) [reply packages/website-frontend/index.html:7]: It’s an .ico file, I copied this from the old implementation 😅

**Pull request merged**